app/assets/stylesheets/application.css.scss

```diff
@@ -91,9 +96,50 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
   }
 
   body {
+    background-color: $bg-color;
     margin: 0 0 20px;
   }
 
+  form {
+    background: $white;
+    border-radius: 5px;
+    border-collapse: inherit;    
+    border: 1px solid $border;
+  }
+
+  #login-box form{
+    border-radius: none;
+    padding: 0;
+    margin: 0;
+    border:none;
+  }
+
+  form {
+    padding: 20px;
+    margin-bottom:20px; 
+    
+    #user_notifications{
+      position: relative;
+      margin: 0;
+    }
+  
+  }
+
+  table.users {
+    tr:nth-of-type(2) th
+    {
+      border-top:0px;
```

Comment:
Colon after property should be followed by one space<br>`0px` should be written without units as `0`